docs/general/ad-filtering/create-own-filters.md

```diff
@@ -316,6 +316,7 @@ If you want the rule not to be applied to certain apps, start the app name with
 
 Apps in the modifier value cannot have a wildcard, e.g. `$app=com.*.music`.
 Rules with such modifier are considered invalid.
+In AdGuard Browser Extension MV3, regexps and any tld domains are not supported.
```

Comment:
"AdGuard Browser Extension MV3" or "AdGuard for Chrome MV3"?

What will we do when it's released (in a couple of weeks) and the main extension will become MV3?

Reply:
@vozersky @AlinaNaletova any thoughts on the naming?

Reply:
I'd say "AdGuard AdBlocker (MV3)" or even "AdGuard AdBlocker" will be fine for Chrome store